lib/players/human.rb

```diff
@@ -0,0 +1,12 @@
+module Players
+
+  class Human < Player
+
+    def move(input)
+      puts "It's your turn #{token}! What's your move? (Enter a number between 1 and 9)"
+      input = gets.strip
```

Comment:
Hi Cernan! Thanks for the comments and edits. I implemented the changes you recommended and pushed the changes to Github.

Thanks!
Callie